extensions/amp-inputmask/0.1/inputmask-custom-alias.js

```diff
@@ -43,81 +52,93 @@ export function factory(Inputmask) {
        * @return {string}
        */
       'onBeforeMask': function(value, opts) {
-        const processedValue = value.replace(/^0{1,2}/, '').replace(/[\s]/g, '');
         const prefixes = opts['prefixes'];
+        const trimZeros = opts['trimZeros'];
+
+        let processedValue = value;
```

Comment:
I tend to like to remove special cases, and just go through one code path. I think this should be equivalent?

```javascript
const trimZeros = opts['trimZeros'] || 0;
const processedValue = value.replace(new RegExp('^0{0,${trimZeros}}'), '').replace(/[\s]/g, '');
```

I think having the code a bit shorter and easier to read outweighs any marginal performance benefits (i.e. from doing a replace that doesn't do anything).